.coveragerc

```diff
@@ -818,6 +818,8 @@ omit =
     homeassistant/components/ozw/services.py
     homeassistant/components/panasonic_bluray/media_player.py
     homeassistant/components/panasonic_viera/media_player.py
+    homeassistant/components/peco/__init__.py
+    homeassistant/components/peco/sensor.py
```

Comment:
a gold requirement would be
>    Full test coverage for the config flow
>    Above average test coverage for all integration modules
>    Tests for fetching data from the integration and controlling it (docs)

I guess that means removing `__init__.py` and `sensor.py` from here

Reply:
Hi there! I’m slowly working on getting code coverage to 100%. Is there any recommendation on how to mock an API?

Reply:
Are you using httpx or aiohttp now? For latter use `from tests.test_util.aiohttp import AiohttpClientMocker` to use a response fixture. I guess there is a httpx equivalent too.

Reply:
I've searched the repository, and it's a little bit confusing. Do I just import it and it automatically has a mock available in each test method

Reply:
I did use it for Fronius tests (https://github.com/home-assistant/core/tree/dev/tests/components/fronius) but this has multiple devices with multiple endpoints. 
See the `fixtures` directory - there are the various responses that are loaded by a helper function in `__init__.py`. 

I guess for a single-endpoint API you will find less complex implementations in other integrations. Just have a search in the codebase.

Reply:
How does `aioclient_mock` get passed into [this](https://github.com/home-assistant/core/blob/dev/tests/components/fronius/test_init.py#L12) method?

Reply:
It's a `pytest.fixture` - see https://docs.pytest.org/en/6.2.x/fixture.html#requesting-fixtures

The scope of this should be code review. For general integration development questions please use HA discord #devs_core or the forums.